bitrafael-client/src/main/java/com/generalbytes/bitrafael/api/wallet/bch/MasterPrivateKeyBCH.java

```diff
@@ -0,0 +1,45 @@
+package com.generalbytes.bitrafael.api.wallet.bch;
```

Comment:
This is just a class skeleton. Please provide the missing proper implementation.